src/main/webapp/js/common/react/display_navbar.js

```diff
@@ -37,6 +37,7 @@ const urls = [
  */
 function initNavbar() {
   const navbarContainer = document.getElementById('navbar-container');
+  navbarContainer.classList.add('sticky-top');
```

Comment:
nice to learn about this class

Reply:
It's super convenient, although it isn't yet [supported in all browsers](https://caniuse.com/#feat=css-sticky). The good news is that there's no real downside if the browser doesn't support it beyond the navbar not sticking to the top of the page.